plugins/vppv2/model/punt/punt.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package punt;
+
+/* L3 protocol definition */
+enum L3Protocol {
+    UNDEFINED_L3 = 0;
+    IPv4 = 4;
+    IPv6 = 6;
+    ALL = 10;
+}
+
+/* IpRedirect allows otherwise dropped packet which destination IP address matching some of the VPP addresses
+to redirect to the defined next hop address via the TX interface */
+message IpRedirect {
+    L3Protocol l3_protocol = 1; /* L3 protocol */
+    string rx_interface = 2;    /* Receive interface name. Optional, only redirect traffic incoming from this interface */
+    string tx_interface = 3;    /* Transmit interface name */
+    string next_hop = 4;        /* Next hop IP where the traffic is redirected */
+}
+
+/* allows otherwise dropped packet which destination IP address matching some of the VPP interface IP addresses to be
+punted to the host. L3 and L4 protocols can be used for filtering */
+message ToHost {
+    L3Protocol l3_protocol = 2;     /* L3 destination protocol a packet has to match in order to be punted */
+    enum L4Protocol {               /* Currently VPP only supports UDP*/
```

Comment:
To avoid having to move this later, when `L4Protocol` type is used in some new punt feature, perhaps it might be better to put it next to `L3Protocol`.